src/ensembl/src/content/app/entity-viewer/EntityViewer.tsx

```diff
@@ -51,7 +51,7 @@ export type EntityViewerParams = {
 };
 
 const client = new ApolloClient({
-  uri: 'http://193.62.55.158:30466'
+  uri: 'http://localhost:5000'
```

Comment:
TODO: This needs to be reverted before merging this PR.

Reply:
Don't forget ;-)